helm/dagger/templates/engine-daemonset.yaml

```diff
@@ -54,9 +58,17 @@ spec:
         - name: dagger-engine
           image: {{ if .Values.engine.image.ref }}{{ .Values.engine.image.ref }}{{ else }}registry.dagger.io/engine:v{{ .Chart.Version }}{{ end }}
           imagePullPolicy: {{ .Values.engine.image.pullPolicy }}
-          {{- if .Values.engine.args }}
+          {{- if or .Values.engine.port .Values.engine.args }}
           args:
+            {{- if .Values.engine.port }}
+            - "--addr"
+            - "tcp://0.0.0.0:{{ .Values.engine.port }}"
+            - "--addr"
+            - "unix:///run/dagger/engine.sock"
```

Comment:
I just loaded the `/run` change which came in effect since 2015: https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard . Good change 👍